docs/flash_instructions.txt

```diff
@@ -0,0 +1,41 @@
+How to put devices into Bootsel mode
+------------------------------------
+- If the device is factory new, it will go into Bootsel mode automatically when connected.
+- If the device has an old firmware installed, there are multiple ways:
+  - Option 1: Use the Ctrl app > Tools > Commands > Restart in bootsel mode.
+  - Option 2: In the controller, hold Home+Select while the controller is connected via USB.
+  - Option 3: In the Alpakka v1, double press the reset button next to the USB port (see manual).
+  - Option 4: In the Alpakka v0, hold the white button in the Raspberry Pico while connecting the USB cable.
+  - Option 5: In the dongle, hold the multifunction button while connecting the USB cable.
```

Comment:
This is not really an option for the device, but for the dongle => move to separate para